layouts/partials/sidebar-block.html

```diff
@@ -0,0 +1,54 @@
+<!-- 
```

Comment:
Why are we splitting this into a separate file? It looks like this was merged into the sidebar file so we shouldn't have this one

Reply:
Had a discussion with Eric, we don't want to change how the default sidebar behaves, it's better to have a separate one. 

The default sidebar does not support children.